File: javascript/Somatorio.js
function somatorio(numeros) {
    return numeros.reduce((soma, num) => soma + num, 0); // No JS podemos usar o reduce para somar arrays facilmente
}
const numeros = [5, 10, 15, 20];
console.log("Soma de " + numeros + ": " + somatorio(numeros));

File: java/Somatorio.java
package java;
import java.util.Arrays;

public class Somatorio {
    public static void main(String[] args) {
        int[] numeros = {5, 10, 15, 20};
        System.out.println("Soma de " + Arrays.toString(numeros) + ": " + somatorio(numeros));
    }

    public static int somatorio(int[] numeros) {
        int soma = 0;
        for (int num : numeros) {
            soma += num;
        }
        return soma;
    }
}
